downloader_library/src/main/java/com/google/android/vending/expansion/downloader/Helpers.java

```diff
@@ -209,54 +215,97 @@ public static String getExpansionAPKFileName(Context c, boolean mainFile, int ve
     }
 
     /**
-     * Returns the filename (where the file should be saved) from info about a
-     * download
+     * Returns the filename (where the file should be saved) from info about a download
      */
     static public String generateSaveFileName(Context c, String fileName) {
-        String path = getSaveFilePath(c)
+        return getSaveFilePath(c)
                 + File.separator + fileName;
-        return path;
     }
 
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
```

Comment:
Check if this is TargetApi is required anymore

Reply:
Removed